From 6cb1397fe7d67a035dfce77a2f81644f86931b50 Mon Sep 17 00:00:00 2001
From: Lionel Debieve <lionel.debieve@foss.st.com>
Date: Wed, 8 Dec 2021 10:40:57 +0100
Subject: [PATCH 0524/1695] drivers: include: add power define in
 stm32mp_dt_bindings

Add the power bindings include in the generic stm32mp include file.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git c6810dd6ecebf16c4a81ae113bc4095f8e69e67c drivers: include: add power define in stm32mp_dt_bindings]
Change-Id: I4c1547bdfa8258d22cc7f5ee08ba694b7f1e8fbe
Signed-off-by: Lionel Debieve <lionel.debieve@foss.st.com>
Signed-off-by: Pascal Paillet <p.paillet@st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/358367
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
---
 core/include/drivers/stm32mp_dt_bindings.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/include/drivers/stm32mp_dt_bindings.h b/core/include/drivers/stm32mp_dt_bindings.h
index f3ebb706a..725234ec5 100644
--- a/core/include/drivers/stm32mp_dt_bindings.h
+++ b/core/include/drivers/stm32mp_dt_bindings.h
@@ -8,6 +8,7 @@
 #ifdef CFG_STM32MP13
 #include <dt-bindings/clock/stm32mp13-clks.h>
 #include <dt-bindings/clock/stm32mp13-clksrc.h>
+#include <dt-bindings/power/stm32mp1-power.h>
 #include <dt-bindings/regulator/st,stm32mp13-regulator.h>
 #include <dt-bindings/reset/stm32mp13-resets.h>
 #include <dt-bindings/soc/stm32mp13-etzpc.h>
-- 
2.47.2

